src/librustc_target/spec/thumbv7a_pc_windows_msvc.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use spec::{LinkerFlavor, Target, TargetOptions, TargetResult, PanicStrategy};
+
+pub fn target() -> TargetResult {
+    let mut base = super::windows_msvc_base::opts();
+
+    // Prevent error LNK2013: BRANCH24(T) fixup overflow
+    base.pre_link_args.get_mut(&LinkerFlavor::Msvc).unwrap().push(
+        "/OPT:NOLBR".to_string());
+
+    base.panic_strategy = PanicStrategy::Abort;
```

Comment:
Could a FIXME be left here to eventually remove this line?

Reply:
Done.